src/Microsoft.Fast.Components.FluentUI/Components/DataGrid/Infrastructure/DisplayAttributeExtesions.cs

```diff
@@ -0,0 +1,33 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Reflection;
+using System.ComponentModel.DataAnnotations;
+
+namespace Microsoft.Fast.Components.FluentUI.DataGrid.Infrastructure;
+internal static class DisplayAttributeExtesions
+{
+
+    public static string? GetDisplayAttributeString(this Type itemType, string propertyName)
+    {
+        PropertyInfo? propertyInfo = itemType.GetProperty(propertyName);
```

Comment:
We need to get rid of these warnings below

Reply:
I can only disable it.
this is a warning for using reflection, so code analysis warn me that 'Trimming may remove property or functions that u need those'
if I add `DynamicallyAccessedMembers` attribute to 'type' parameter of `GetDisplayAttributeString`, warning will resolved. but I get the same warning on `OnParameterSet` function (the caller function). and we cannot add the same attribute to overrides function and the warning will be remain.

on the other hand, model is not in our assembly and we are not worry about triming models in our project
if you agree, I disable it

Reply:
Let's fix the warning in the `GetDisplayAttributeString` method (by adding the `DynamicallyAccessedMember` attribute) and add the following attribute to the `OnParameterSet` method:
```
[System.Diagnostics.CodeAnalysis.SuppressMessage("Trimming", "IL2072:Target parameter argument does not satisfy 'DynamicallyAccessedMembersAttribute' in call to target method. The return value of the source method does not have matching annotations.", Justification = "'DynamicallyAccessedMembersAttribute' cannot be added to overriden method")]
```